### PR TITLE
Enable edge.country_crossing to be traced

### PR DESCRIPTION
**Pull request by johannes-no**
## Issue 
For a better accuracy of our services we need more precise information about country crossings.

## Fix
With this pull request we enable the "trace_attributes" endpoint to retrieve information about a country crossings at edge level.

The code adaptations are symmetrical to example for the attribute "edge.road_class".

## Tasklist

 - [x] Update the docs with any new request parameters or changes to behavior described
 - [x] Update the [changelog](CHANGELOG.md)

Johannes Nonnenmacher [johannes.nonnenmacher@mercedes-benz.com](mailto:johannes.nonnenmacher@mercedes-benz.com), Mercedes-Benz Tech Innovation GmbH
[Provider Information](https://github.com/mercedes-benz/foss/blob/master/PROVIDER_INFORMATION.md)

Licensed under MIT

### CODE REVIEW

**Review by chrstnbwnkl — CHANGES_REQUESTED** [review]
Would be nice to add it to the JSON as well, should be a one-liner!

**Comment by johannes-no** [comment]
> Would be nice to add it to the JSON as well, should be a one-liner!

Sorry which json do you mean?

**Review by chrstnbwnkl — APPROVED** [review]

File: proto/trip.proto [hunk 1]
```diff
@@ -180,6 +180,7 @@ message TripLeg {
     repeated StreetName tunnel_name = 55;
     float elevation_sampling_interval = 56;
     repeated float elevation = 57;
+    bool ctry_crossing = 58;
```
Comment (kevinkreiser) [inline proto/trip.proto:183]: just one nitpick, can you please spell it out? none of the other ones are abbreviated

Comment (johannes-no) [reply proto/trip.proto]: Done

**Review by kevinkreiser — APPROVED** [review]

**Pull request merged**